### ISSUE TITLE
glitch

### ISSUE THREAD

**Issue by robloxforlifexo** [lanylow/rbxfpsunlocker-osx#305]
whenever I join a game my fps keeps going down and up and the game that I'm playing (the buildings) just start flashing and glitching how do I fix this?

**Comment by juliuwulooooolll** [comment]
Same here everytime I join a game the buildings in game start flashing and my fps go up and down it's really laggy.

**Comment by robloxforlifexo** [comment]
how do I fix this?

**Comment by robloxforlifexo** [comment]
> Same here everytime I join a game the buildings in game start flashing and my fps go up and down it's really laggy.

have u fixed it yet?

**Comment by juliuwulooooolll** [comment]
Yes I had deleted the fps unlocker on the mac account I was using and made a new mac user and just redownloaded and redid everything and it works perfectly now.
